lib/matplotlib/path.py

```diff
@@ -558,14 +558,12 @@ def intersects_bbox(self, bbox, filled=True):
         :class:`~matplotlib.transforms.Bbox`.
 
         *filled*, when True, treats the path as if it was filled.
-        That is, if one path completely encloses the other,
-        :meth:`intersects_path` will return True.
+        That is, if the path completely encloses the bounding box,
+        :meth:`intersects_bbox` will return True.
+
+        The bounding box is always considered filled.
```

Comment:
It's a change in behaviour when filled=False, however this mode was only used in ```legend.py```, and in that context the old behaviour was incorrect: the bounding box (legend) is always filled even when the plot lines are not filled.